backend/src/ee/models/gitRisks.ts

```diff
@@ -26,15 +29,12 @@ export type GitRisks = {
   ruleID: string;
   fingerprint: string;
   fingerPrintWithoutCommitId: string
-
-  isFalsePositive: boolean; // New field for marking risks as false positives
-  isResolved: boolean; // New field for marking risks as resolved
   riskOwner: string | null; // New field for setting a risk owner (nullable string)
```

Comment:
I see you added a new field but it isn't being used anywhere?

Reply:
This is old - before I worked on it. I removed it to use the specific risk statuses